students/MikeShand/lesson03/mailroom.py

```diff
@@ -1,51 +1,51 @@
 #!/usr/bin/env python3
 
+donors = [['Andy', [10.00]], ['Bill', [15.00, 25.00]], ['Chuck', [20.00, 30.00, 40.00]]]
 
 
-donors=[['Andy', [10.00]],['Bill',[15.00, 25.00]],['Chuck',[20.00,30.00,40.00]]]
-
 def donor_list():
-    donor_list=[]
     for x in donors:
         print(x[0])
     thank_you()
 
 
 def thank_you():
-    print('Please enter the donor name\n (Type "list" for a list of current donor names)\n Press "q" to return to console')
-    cd=input(':')
+    print('Please enter the donor name\n (Type "list" for a list of current donor names)\n '
+          'Press "q" to return to console')
+    cd = input(':')
 
-    if cd.lower()=='list':
+    if cd.lower() == 'list':
         donor_list()
 
-    elif cd.lower()=='q':
-        console()   
+    elif cd.lower == 'q':
+        return
 
     else:
 
-        amount=float(input('Please enter the donation amount:'))
+        amount = float(input('Please enter the donation amount:'))
 
-        for j,k in enumerate(donors):
-            if cd==k[0]:
+        for j, k in enumerate(donors):
+            if cd == k[0]:

```

Comment:
I would void `j` and `k` vars as they are ambiguous, and I would also not use `enumerate` here. 

You don't need the index, you can just loop over: `for donor in donors` since you end up accessing donor by index later on anyway.